src/app/shared/json-ld/json-ld.component.spec.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * MIT License
```

Comment:
Whether or not to address this is up to you:
I'm a little bit concerned that, absent our context, it's not clear what these two license blocks mean, in terms of exactly what and how we're licensing.  Maybe we could use the comment below the imports as a starting point, and say something like:

```
The following code was adapted from https://github.com/coryrylan/ngx-lite, 
as that repo and its corresponding NPM releases have not kept up with the latest Angular releases.
The original code was licensed under the included MIT license, and we sublicense the derived code
under the included Apache license.  The included MIT license does not apply to other source files,
except where noted.
```

Reply:
That's better said and more explicit than what I have... Good suggestion.